qiita_db/util.py

```diff
@@ -969,14 +969,21 @@ def get_processed_params_tables():
         return TRN.execute_fetchflatten()
 
 
-def get_lat_longs():
+def get_lat_longs(portal):
```

Comment:
The API shouldn't change, the portals is something that is transparent to most of the API. Check other parts of the code (the Study class, user, etc...) to see how the portals stuff is handled completely transparent from the API point of view.